### PR TITLE
#62 - spacing in brackets

### LINKED ISSUE #62: Spacing in brackets is not checked
Things like: 
```php
function ( $user, $password): void {
```

and this:
```php
if ( $whatever ) {

}
```

and this:
```php
        if ($status === PasswordBroker::RESET_LINK_SENT)
        {
```

are not checked and not fixed. There's probably some fixer missing after ECS dropping in #52.

### PR DESCRIPTION
**Pull request by Baakoma**
It should close #62

Before:
```php
<?php

class BracesFixer
{
    public function getNameLabel( string $name, ?string $title = null ): string {
        $label = $name;
        if ( $title !== null )
        {
            $label .= " " . $title;
        }

        return $label;
    }
}
```

After:
```php
<?php

declare(strict_types=1);

class BracesFixer
{
    public function getNameLabel(string $name, ?string $title = null): string
    {
        $label = $name;
        if ($title !== null) {
            $label .= " " . $title;
        }

        return $label;
    }
}

```

### CODE REVIEW

File: tests/codestyle/fixtures/bracesFixer/expected.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
```
Comment (jsawo) [inline tests/codestyle/fixtures/bracesFixer/expected.php:3]: as I understand it, this PR deals with fixing braces formatting around `{` and parentheses `(`.
This expected/actual pair seems to suggest that we're also adding `declare(strict_types=1);` and changing `string` into `?string`

File: tests/codestyle/fixtures/noUselessParenthesisFixer/actual.php [hunk 2]
```diff
@@ -2,7 +2,6 @@
 
 declare(strict_types=1);
 
-if (($a === 1))
-{
+if (($a === 1)) {
```
Comment (krzysztofrewak) [inline tests/codestyle/fixtures/noUselessParenthesisFixer/actual.php:5]: Why did you change this?

**Review by jsawo — CHANGES_REQUESTED** [review]

File: tests/codestyle/fixtures/noUselessParenthesisFixer/actual.php (hunk 2 above)
Comment (Baakoma) [reply tests/codestyle/fixtures/noUselessParenthesisFixer/actual.php:5]: Because this fixer is responsible for fixing a different thing.

**Review by krzysztofrewak — APPROVED** [review]

**Review by jsawo — APPROVED** [review]
